README.md

```diff
@@ -1,15 +1,53 @@
 # ForemanPuppet
 
-Adds Puppet External node classification funcionality for [Foreman](theforeman.org).
+[![Gem Version](https://img.shields.io/gem/v/foreman_puppet.svg)](https://rubygems.org/gems/foreman_puppet)
+[![Code Climate](https://codeclimate.com/github/theforeman/foreman_puppet/badges/gpa.svg)](https://codeclimate.com/github/theforeman/foreman_puppet)
```

Comment:
@ezr-ondrej do we want such a code climate badge? I've seen them in foreman_remote_execution and some other plugins.
I guess this badge needs to be configured in some way or form?

Reply:
we would need to allow code climate for this repository. I'm ok with that but that's more of a "do developers for this repo want to be held to higher code quality standards". I would like the answer to be yes, but I don't see it necessary :)

Reply:
I'll remove the badge for now :)